homeassistant/components/api.py

```diff
@@ -37,13 +37,14 @@ def setup(hass, config):
     # /api - for validation purposes
     hass.http.register_path('GET', URL_API, _handle_get_api)
 
+    # /api/config
+    hass.http.register_path('GET', URL_API_CONFIG, _handle_get_api_config,
```

Comment:
I prefer this to be a separate API call. Maybe `/api/discovery_info` ? That way it cannot go wrong with auth. Like we do not have to worry about it at all.